engine/labels.go

```diff
@@ -34,6 +30,10 @@ func createLabel(e *Env, labelName *string, label *PadLabel) error {
 
 	var labelColor *string
 	if label.Color != "" {
+		if strings.HasPrefix(label.Color, "#") {
+			label.Color = label.Color[1:]
+		}
+
```

Comment:
We can use the `TrimPrefix` that will remove the "#" if it exists, otherwise it remains unchanged. 

```suggestion
		label.Color = strings.TrimPrefix(label.Color, "#")
```

Reply:
Oh, this is what I get for only having very rough knowledge of the Go standard library. Had no idea `TrimPrefix` existed. Committing that change! 🙌🏼